src/Parser/EnhancementShaman/Modules/ShamanCore/DamageDone.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Icon from 'common/Icon';
+import { formatNumber, formatPercentage } from 'common/format';
+
+import CoreDamageDone from 'Parser/Core/Modules/DamageDone';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class DamageDone extends CoreDamageDone {
+  suggestions(when) {
```

Comment:
This suggestion would be better placed in the Always Be Casting module where it's also calculated.

Reply:
Damage Done doesn't appear to be computed within ABC, leaving this within it's own module for now

Reply:
This suggestion isn't about damage done, but about casting time so I think it would better fit the ABC module.